fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentUtils.scala

```diff
@@ -136,7 +136,7 @@ object FaciaContentUtils {
   )
   def maybeShortUrl(fc: FaciaContent) = fieldsGet(fc)(_.flatMap(_.shortUrl))
   def shortUrl(fc: FaciaContent): String = maybeShortUrl(fc).getOrElse("")
-  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replace("http://gu.com", ""))
+  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replaceFirst("^[a-zA-Z]+://gu.com", ""))
```

Comment:
@dominickendrick it would, but I noticed in other places in frontend we use what I wrote, so I just copied and pasted that, but yeah, what everyone else said